contribute.md

```diff
@@ -59,6 +59,16 @@ If you experience problems related to therubyracer or libv8, you can install wit
 1. Run all the tests: `rake test`
 1. Or a single test: `ruby -Itest test/functional/your_test.rb` (if you didn't run all tests before, you need to prepare the test environment with `rake db:test:prepare` the first time).
 
+#### A note on Rubies
```

Comment:
That feels a more informal style than the rest of the doc, how about "#### Supported Ruby versions" ?